ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts.hbs

```diff
@@ -26,7 +26,21 @@
             <p>{{host.description}}</p>
           </row.headerCell>
           <row.cell>
-            <Rose::Badge>{{host.type}}</Rose::Badge>
+            {{#if host.isStatic}}
+              <Rose::Badge>
+                {{t (concat 'resources.host.types.' host.compositeType)}}
```

Comment:
Host sets and hosts can reuse the same type translations used by host catalogs, to reduce duplication.

Reply:
do you recommend removing the `types` translations from host/host-set?  
perhaps move the types to the form translation file?

Reply:
Since we already have these `types` specified for host catalogs, I'd remove mention of them from hosts and host sets and change the references to point to host catalogs:  https://github.com/hashicorp/boundary-ui/blob/main/addons/core/translations/resources/en-us.yaml#L87

Reply:
sounds good!

Reply:
Host sets and hosts will always have the same set of types as host catalogs.